lua/telescope/pickers.lua

```diff
@@ -383,6 +388,26 @@ function Picker:find()
   self.prompt_prefix = prompt_prefix
   self:_reset_prefix_color()
 
+  local results_win, results_opts = popup.create('', popup_opts.results)
+  local results_bufnr = a.nvim_win_get_buf(results_win)
+
+  self.results_bufnr = results_bufnr
+  self.results_win = results_win
+
+  -- TODO: Should probably always show all the line for results win, so should implement a resize for the windows
+  a.nvim_win_set_option(results_win, 'wrap', false)
+  a.nvim_win_set_option(results_win, 'winhl', 'Normal:TelescopeNormal')
+  a.nvim_win_set_option(results_win, 'winblend', self.window.winblend)
+
+  local results_border_win
+  if not self.window.externalize then
+    results_border_win = results_opts.border and results_opts.border.win_id
+    if results_border_win then
+      vim.api.nvim_win_set_option(results_border_win, 'winhl', 'Normal:TelescopeResultsBorder')
+    end
+  end
```

Comment:
Why was this whole part moved down. Previously on 340

Reply:
This is so that the prompt, result, and preview windows will be placed in the default display position in telescope.nvim when the external window layout process is done in the GUI. 
If this part is not moved, prompt will be displayed on top. However, since this is a small minor issue, and is strongly based on the layout process of goneovim, it may be better to revert it back.

Reply:
Oh interesting i didn't know that (but i also have never used a GUI). We can keep it :)

But does that mean that option `prompt_position = 'top'` doesn't work?

Reply:
Yes, `prompt_position`, etc. do not work. This is because, as far as I know, there is no way to directly specify the position of the external window in the current neovim implementation. 
One way, as mentioned at the beginning, is to display them as a normal float window once before externalizing them, and then externalize them instantly. However, I have not been able to investigate how to get this to work.